spec/exchanges/novaexchange/integration/market_spec.rb

```diff
@@ -29,21 +29,4 @@
     it { expect(@ticker.timestamp).to_not be nil }
     it { expect(@ticker.payload).to_not be nil }
   end
-
```

Comment:
i noticed that too and issued a commit for that. Just noticed the discussion here later on